### PR TITLE
Allow query_id override, remove unnecessary examples

### PR DESCRIPTION
**Pull request by slvrtrn**
## Summary
* `query_id` can be now overridden for all main client's methods: `query`, `exec`, `insert`.
* Superfluous code examples were removed

## Checklist
- [x] Unit and integration tests covering the common scenarios were added
- [x] A human-readable description of the changes was provided to include in CHANGELOG

### CODE REVIEW

File: examples/stream_created_from_array_json.ts [hunk 1]
```diff
@@ -1,38 +0,0 @@
-import { createClient } from '@clickhouse/client'
```
Comment (slvrtrn) [inline examples/stream_created_from_array_json.ts:1]: I think in this case stream is pretty much useless and just an array could be used (and we have these examples already)

File: examples/insert_json_stream.ts [hunk 2]
```diff
@@ -1,44 +0,0 @@
-import { createClient } from '@clickhouse/client'
```
Comment (slvrtrn) [inline examples/insert_json_stream.ts:1]: "endless" stream examples have broader coverage

File: examples/stream_created_from_array_raw.ts [hunk 3]
```diff
@@ -1,32 +0,0 @@
-import { createClient } from '@clickhouse/client'
```
Comment (slvrtrn) [inline examples/stream_created_from_array_raw.ts:1]: same as with `stream_created_from_array_json.ts`

File: __tests__/integration/select.test.ts [hunk 4]
```diff
@@ -321,17 +332,6 @@ describe('select', () => {
...
-      it.skip('returns queryId in response', async () => {
-        const rs = await client.query({
-          query: 'SELECT number FROM system.numbers LIMIT 5',
-          format: 'JSON',
-        })
-
-        const response = await rs.json<ResponseJSON<{ number: string }>>()
-
-        expect(response.query_id).toBeInstanceOf(Number)
-      })
```
Comment (slvrtrn) [inline __tests__/integration/select.test.ts:334]: This is an outdated version of the same test above

File: examples/stream_created_from_array_raw.ts (hunk 3 above)
Comment (mshustov) [reply examples/stream_created_from_array_raw.ts:1]: That's the only example of inserting serialized data. I'd keep it

**Review by mshustov — APPROVED** [review]

**Pull request merged**